static/js/app.js

```diff
@@ -1012,7 +1012,15 @@ $(function () {
                 age_groups += "18,";
             }
 
-            return (age_groups ? age_groups.slice(0, -1) : "0");
+            if (age_groups == "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,") {
```

Comment:
Maybe something like this would be better?

    Array.from(new Array(18), (x,i) => i + 1).join() + ","

Reply:
Yeah, I'll change it to something less hackish.